pkg/jobs/slack_worker.go

```diff
@@ -0,0 +1,70 @@
+package jobs
+
+import (
+	"context"
+	"github.com/riverqueue/river"
+)
+
+// SlackArgs for the Slack worker to process the job
+// Channel can be a name or ID
+// Message is the text to send
+// DevMode mocks the request
+type SlackArgs struct {
+	Channel string `json:"channel"`
+	Message string `json:"message"`
+	DevMode bool   `json:"dev_mode"`
+}
```

Comment:
Same here, but these above each variable
```suggestion
// SlackArgs for the Slack worker to process the job
type SlackArgs struct {
	// Channel can be a name or ID
	Channel string `json:"channel"`
	// Message is the text to send
	Message string `json:"message"`
	// DevMode mocks the request
	DevMode bool   `json:"dev_mode"`
}
```